lib/eunit/src/eunit_lib.erl

```diff
@@ -574,6 +574,17 @@ trie_match([], []) ->
 trie_match([], _T) ->
     no.
 
+%% ---------------------------------------------------------------------
+%% Convert a string of unknown encoding to a binary
+
+try_to_binary(S) when is_list(S) ->
```

Comment:
You are ensuring that everything everywhere is using unicode above and then expect them to be in latin1?

Lists are supposed to be unicode codepoints in erlang  (nowadays) so this function seems backward to me.